Say Their Names/Source/Controller/Home/Person/PersonDetailsController.swift

```diff
@@ -41,12 +40,13 @@ extension PersonDetailsController: UICollectionViewDataSource, UICollectionViewD
     }
     
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "PersonPhotoCell", for: indexPath)
+        let cell: PersonPhotoCell = collectionView.dequeueCell(for: indexPath)
+        cell.personImage = #imageLiteral(resourceName: "man-in-red-jacket-1681010")
```

Comment:
I don’t think this has been addressed in the project yet but we shouldn’t use image literals as they can cause the app to crash if the asset is removed. Can we use UIImage(named:) instead?